memory-bank/01-project-overview.md

```diff
@@ -0,0 +1,93 @@
+# Apache Traffic Server - Project Overview
+
+## Project Overview
+
+**Apache Traffic Server (ATS)** is a high-performance, open-source HTTP/HTTPS caching proxy server and web application acceleration platform. It's designed as a building block for cloud services and large-scale web applications.
+
+### Key Characteristics
+- **Version**: 10.2.0 (current)
```

Comment:
Should this be the current released version?  Are we going to update this per branch per release?